### PR TITLE
general-ui-fixes-1  #174 #197

### LINKED ISSUE #174: frontend fixes / improvements i will do on current main branch
- [x] about modal : if using icons : use fusion icons  "twitter", "github", "telegram".
- [x] fonts to small compared to richard-ui3 branch ( i think i can find a good middleground between current and planed )
- [x] remove all that css in App.vue ( compare with richard-ui3 branch ) ( we are using full width, grids . no text-align:center )
- [x] identicon in nav is not vertical centered ( probably because a margin-bottom / div / no display block on img )
- [x] style flash notifications . how to trigger one to see ?
- [x] create and use src/assets/placeholder_grant.svg + placeholder_round.svg on not loaded / 404 images

- [x] apply richard-ui3 to cart ( working on it - have to talk with matt about baseinput baseselect for cleaner ui ) ( started on that in branch "cart-with-richard-ui2"  ( what should be a 3 )

### LINKED ISSUE #197: Update Website Metadata
- [ ] Title(s)
- [ ] Favicon
- [ ] Twitter/Social Cards

### PR DESCRIPTION
**Pull request by scco**
related to issue 
- enhanced in LayoutHeader.vue the look of unsupported Networks
- removing hidden little bottom space from jazzicon in LayoutHeader
- removed text-red-600 from index.css as we have text-pink
- changed responsive fontsizes with a compromise for not become to big again
- adding a placeholder_round.svg to our assets ( same as grant for now ... just to have something to embed as placeholder )
- adding basic meta fields to the index.html
- favicon / twitter / open graph meta tags

Closes #174
Closes #197

REMINDER : In index.html ( Line 13-24 )
- have to put real content in for keywords, cards, texts 
- need to link to an absolute https:// url for the dgrants_social_preview.png
- need to edit the real https:// address of our project

### CODE REVIEW

File: app/src/assets/placeholder_round.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<svg width="1920" height="1080" viewBox="0 0 1920 1080" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (mds1) [inline app/src/assets/placeholder_round.svg:1]: Is this the right SVG file? If so what exactly is it for? It seems nearly identical to `placeholder_grant.svg` 
image

File: app/index.html [hunk 2]
```diff
@@ -1,18 +1,43 @@
...
+
+    <meta property="og:title" content="dGRANTS – xxx" />
+    <meta property="og:description" content="xxx" />
+    <meta property="og:image" content="dgrants_social_preview.png" />
+    <meta property="og:url" content="https://xxx" />
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="@gitcoin" />
+    <meta name="twitter:title" content="dGRANTS – xxx" />
+    <meta name="twitter:description" content="xxx" />
+    <meta name="twitter:image" content="dgrants_social_preview.png" />
```
Comment (mds1) [inline app/index.html:24]: > REMINDER : In index.html ( Line 13-24 )
> 
> - have to put real content in for keywords, cards, texts
> - need to link to an absolute https:// url for the dgrants_social_preview.png
> - need to edit the real https:// address of our project

@phutchins Do you have any idea of what the official URL will be yet? Also @scco do you think we can get the real data for this placeholders as part of this PR, or should it be a different PR with an issue to track it?

**Review by mds1 — APPROVED** [review]

**Comment by thelostone-mc** [comment]
@scco do you wanna screenshot for these changes to make it easier to review as well ?

**Review by thelostone-mc — APPROVED** [review]
